lib/server.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+var express = require('express')
+var bodyParser = require('body-parser')
+var routes = require('./routes')
+var http = require('http')
+var https = require('https')
+var errors = require('./errors')
+var logger = require('winston')
+var expressWinston = require('express-winston')
+var extension = require('./extension')
+
+var server, port
+
+function createServer (config, callback) {
+  if (server) return callback(errors.getError('SERVER_ALREADY_CREATED'))
+
+  extension.loadConfiguration(config, function (err) {
+    if (err) return callback(err)
+    // Set default to Infinity
+    http.globalAgent.maxSockets = config.maxSockets || Infinity
+    https.globalAgent.maxSockets = config.maxSockets || Infinity
+
+    var app = express()
+
+    app.use(bodyParser.json({limit: '10mb'}))
+
+    if (config.winstonInstance) logger = config.winstonInstance
+
+    var expressLogConfig = {
+      winstonInstance: logger,
+      msg: '{{res.statusCode}} HTTP {{req.method}} {{req.url}} {{res.responseTime}}ms',
+      meta: false
+    }
+
+    app.use(expressWinston.logger(expressLogConfig))
+    app.use(expressWinston.errorLogger(expressLogConfig))
+
+    // hijack console.log
```

Comment:
Lets remove this code to hijack console.log. I meant to say the other day that whoever wants to use this can pass in a logger that hijack's the logs. We don't need to do it as part of the core code.